labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -58,24 +63,23 @@ object CollectionExercise02 {
 
 object CollectionExercise03 {
   /**
-   * Create a method that checks that each subsequent value is greater than
-   * the previous one.
-   * E.g.:
-   * checkValuesIncrease(Seq(1,2,3)) == true
-   * checkValuesIncrease(Seq(1,2,2)) == false
-   */
+  * Create a method that checks that each subsequent value is greater than
+  * the previous one.
+  * E.g.:
+  * checkValuesIncrease(Seq(1,2,3)) == true
+  * checkValuesIncrease(Seq(1,2,2)) == false
+  */
   def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+    seq == seq.sorted
```

Comment:
I don't think this will work - it will return true for (1, 2, 2) although apparently none of the tests checked that even though it was in the function header...